src/uu/cksum/cksum.md

```diff
@@ -0,0 +1,23 @@
+# cksum
+
+```
+cksum [OPTIONS] [FILE]...
+```
+
+Print CRC and size for each file
+
+## After Help
+
+DIGEST determines the digest algorithm and default output format:
+
+-a=sysv:    (equivalent to sum -s)
+-a=bsd:     (equivalent to sum -r)
+-a=crc:     (equivalent to cksum)
+-a=md5:     (equivalent to md5sum)
+-a=sha1:    (equivalent to sha1sum)
+-a=sha224:  (equivalent to sha224sum)
+-a=sha256:  (equivalent to sha256sum)
+-a=sha384:  (equivalent to sha384sum)
+-a=sha512:  (equivalent to sha512sum)
+-a=blake2b: (equivalent to b2sum)
+-a=sm3:     (only available through cksum)
```

Comment:
Can you please turn the list into a markdown list (see for example [stat](https://github.com/uutils/coreutils/blob/main/src/uu/stat/stat.md))? Otherwise the [rendering](https://github.com/uutils/coreutils/blob/7c6a948526f520903f5d502178e494a308abdfb8/src/uu/cksum/cksum.md) doesn't look that great ;-)